src/components/utils/field-constants.js

```diff
@@ -259,6 +259,8 @@ export const CONNECT = 'connect';
 export const CONTAINER_ID = 'containerId';
 export const CONTAINER_NAME = 'containerName';
 
+export const REACTIVE_SLACKS_THRESHOLD = 'reactiveSlacksThreshold';
```

Comment:
This is only used in voltage-init components so for me it should be defined there

Reply:
moved to a new voltage-init-constants.ts file